{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "PowerVM Hypervisor",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "FW940"
                        },
                        {
                            "status": "affected",
                            "version": "FW950"
                        },
                        {
                            "status": "affected",
                            "version": "FW860"
                        },
                        {
                            "status": "affected",
                            "version": "FW1010"
                        }
                    ]
                }
            ],
            "datePublic": "2021-12-02T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "IBM PowerVM Hypervisor FW860, FW940, FW950, and FW1010, through a specific sequence of VM management operations could lead to a violation of the isolation between peer VMs. IBM X-Force ID: 210019."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "NONE",
                        "baseScore": 8.6,
                        "baseSeverity": "HIGH",
                        "confidentialityImpact": "HIGH",
                        "exploitCodeMaturity": "UNPROVEN",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "NONE",
                        "remediationLevel": "OFFICIAL_FIX",
                        "reportConfidence": "CONFIRMED",
                        "scope": "CHANGED",
                        "temporalScore": 7.5,
                        "temporalSeverity": "HIGH",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.0/I:N/AC:L/AV:N/UI:N/C:H/A:N/S:C/PR:N/RC:C/RL:O/E:U",
                        "version": "3.0"
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Obtain Information",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2022-01-05T16:30:11",
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/6525032"
                },
                {
                    "name": "ibm-powervm-cve202138918-info-disc (210019)",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_XF"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/210019"
                }
            ]
        }
    },
    "cveMetadata": {
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "assignerShortName": "ibm",
        "cveId": "CVE-2021-38918",
        "dateReserved": "2021-08-16T00:00:00",
        "dateUpdated": "2022-01-05T16:30:11",
        "state": "PUBLISHED",
        "datePublished": "2022-01-05T16:30:11.969607Z"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}
